buildlib/fixup-include/rdma-vmw_pvrdma-abi.h

```diff
@@ -157,6 +159,8 @@ struct pvrdma_resize_cq {
 
 struct pvrdma_create_srq {
 	__u64 buf_addr;
+	__u32 buf_size;
+	__u32 reserved;
 };
 
```

Comment:
What is up with this kernel uAPI change? If srq was not supported until now why did a create_srq  struct exist?

Reply:
That is a mistake on our part--we should not have had a `pvrdma_create_srq` struct before this.

Is it preferable in this situation to create a `pvrdma_create_srq_v2`? We also noticed that we didn't update the ABI version either, which we'd need to do if we went with the above approach.

Reply:
It looks like it was never used in the kernel at all, so it should be OK to change it however you like.

You should not need to update the ABI because the kernel create_srq call will currently return failure and this is good enough.